src/metabase/email/alert.mustache

```diff
@@ -1,7 +1,6 @@
 {{> metabase/email/_header}}
-    <p><a href="{{questionURL}}">{{questionName}}</a> has {{alertCondition}}.</p>
```

Comment:
Was this intentional?

Reply:
Yes, this was the main purpose of the PR: with the addition of actual tabular results, the paragraph was looking really redundant.

Reply:
Ok, I'll need to change the tests to not look for that anymore and see if I can remove some code that populates that alertCondition